### DISCUSSION TITLE
html в hint

### DISCUSSION THREAD

**Discussion by MadZee-M** [moonshine-software/moonshine#930] [Ideas]
Мне нужно вывести снизу textarea некоторую информацию.
Но в итоге получаю экранированный html.

> Посмотрел в исходниках. Можно решить изменив его на {!! ... !!}
> 
> ![2024-05-07_16-46](https://github.com/moonshine-software/moonshine/assets/32099597/65098a48-fe3b-4398-8316-c64f973b9cd1)

**Comment by lee-to** [comment]
Можете предложить pr

**Comment by lee-to** [comment]
Готово
